src/resources/base.js

```diff
@@ -153,20 +153,24 @@ export default api => (
       } catch (e) {
         return Promise.reject(e);
       }
-
       try {
         const data = this.constructor.wrapJSON(this.toJSON());
-
+        let output;
         let res;
 
         if (this.id) {
           res = await api.put(`${this._url || this.constructor._url}/${this.id}`, { body: data });
+          output = this;
+        } else if (this.constructor._name === 'Rating') {
```

Comment:
constructor.wrapJson makes the input "rating : {input}", but we just need "{input}"

if I set it to this.body, when printing out the rating call, it gives me the addresses, parcels and then ratings, but the way its supposed to return is just ratings without the address and parcel